submarine-workbench/workbench-web-ng/package.json

```diff
@@ -49,14 +53,16 @@
     "codelyzer": "^5.1.2",
     "dotenv": "^8.0.0",
     "https-proxy-agent": "^2.2.1",
+    "husky": "^4.3.0",
     "jasmine-core": "~3.4.0",
     "jasmine-spec-reporter": "~4.2.1",
     "karma": "~4.1.0",
     "karma-chrome-launcher": "~2.2.0",
-    "karma-firefox-launcher": "~1.3.0",
     "karma-coverage-istanbul-reporter": "~2.0.1",
+    "karma-firefox-launcher": "~1.3.0",
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "^1.4.0",
+    "lint-staged": "^10.4.0",
```

Comment:
Thank you for your contribution! I am wondering why we need to remove "lint-staged" from "dependencies" and add it to "devDependencies"?

Reply:
Because code formatter is only used in the development stage. In production, we do not need to run formatter.

Reply:
lint-staged is a package by which you can some check (in here, is prettier) to staged files.